lib/manager/bazel/extract.js

```diff
@@ -196,6 +216,21 @@ function extractPackageFile(content) {
       dep.lookupName = dep.repo;
       dep.lookupType = 'releases';
       deps.push(dep);
+    } else if (
+      depType === 'container_pull' &&
+      currentValue &&
+      digest &&
+      repository &&
+      registry
+    ) {
+      dep.currentDigest = digest;
+      dep.currentValue = currentValue;
+      dep.purl = registry + '/' + repository;
+      dep.depName = depName;
+      dep.repo = repository;
+      dep.datasource = 'docker';
+      dep.lookupName = dep.repo;
```

Comment:
I think we can perhaps skip setting `dep.repo` if it is not used anywhere after. We can probably just use `depName` and `lookupName`.

Reply:
Done.

Reply:
```javascript
dep.datasource = 'docker';
```
cannot be skipped.

Reply:
Agreed that dep.datasource needs to stay.